cmake/OpenCVUtils.cmake

```diff
@@ -808,7 +808,7 @@ macro(ocv_check_modules define)
     pkg_check_modules(${define} ${ARGN})
   endif()
   if(${define}_FOUND)
-    set(HAVE_${define} 1)
+    set(HAVE_${define} TRUE)
```

Comment:
Why?

Reply:
I'll revert this change, it doesn't make any difference, a boolean value just looks clearer to me.